ui/src/components/directory/landingPage/PieChartDashCard.vue

```diff
@@ -44,7 +46,7 @@ function setChartSize(): void {
   const reportTable = document.getElementsByClassName("report-table")[0] as HTMLElement;
   const container = document.getElementById(props.id) as HTMLElement;
   if (!container) {
-    LoggerService.error(undefined, `Failed to set chart size for element id: ${props.id}`);
+    log.error(() => `Failed to set chart size for element id: ${props.id}`);
```

Comment:
does that need to be within a () => 'a' instead of just a string?

Reply:
Examples on the repo show it as arrow function. Not sure if necessary though as might just be building the string.